helix-core/src/main/java/org/apache/helix/tools/ClusterVerifiers/ClusterLiveNodesVerifier.java

```diff
@@ -24,18 +24,24 @@
 import java.util.List;
 import java.util.Set;
 
-import org.apache.helix.zookeeper.api.client.HelixZkClient;
+import org.apache.helix.zookeeper.api.client.RealmAwareZkClient;
+
 
 public class ClusterLiveNodesVerifier extends ZkHelixClusterVerifier {
 
   final Set<String> _expectLiveNodes;
 
-  public ClusterLiveNodesVerifier(HelixZkClient zkclient, String clusterName,
+  public ClusterLiveNodesVerifier(RealmAwareZkClient zkclient, String clusterName,
```

Comment:
Same here, shall we replace the existing constructor with the builder constructor?

Reply:
Will deprecate.